functions/slack/pom.xml

```diff
@@ -155,6 +154,20 @@
 
   <build>
     <plugins>
+      <plugin>
+        <!--
+          Google Cloud Functions Framework Maven plugin
+
+          This plugin allows you to run Cloud Functions Java code
+          locally. Use the following terminal command to run a
+          given function locally:
+
+          mvn function:run -Drun.functionTarget=your.package.yourFunction
```

Comment:
Should also be in README's.

Reply:
Added to parent-level README.

(There are no per-sample "child" READMEs - happy to discuss if that's something you want.)